patches/chrome-browser-ui-webui-webui_util.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/webui/webui_util.cc b/chrome/browser/ui/webui/webui_util.cc
+index fa4038374f7125ad2cb308234fbd8a2afed96bda..95530742da8295c04f6d4b61abef398c887bb6a5 100644
+--- a/chrome/browser/ui/webui/webui_util.cc
++++ b/chrome/browser/ui/webui/webui_util.cc
+@@ -15,7 +15,7 @@ namespace {
+ 
+ void SetupPolymer3Defaults(content::WebUIDataSource* source) {
+   source->OverrideContentSecurityPolicyScriptSrc(
```

Comment:
this can be done with chromium_src overrides for SetupWebUIDataSource and SetupBundledWebUIDataSource

Reply:
Addressed in `25a15fa90dd2bff359b335cd34982e600aa5ad41`